Dockerfile

```diff
@@ -1,19 +1,20 @@
 # Inherit from Heroku's stack
-FROM heroku/heroku:18
+FROM heroku/heroku:22
```

Comment:
can you also use the env var here?
If you define it before this.

Reply:
Is that possible?

Reply:
This is not possible, it's not allowed to define an env variable before the from.
I can define an arg there, and that works. But I cannot use that same variable within the FROM block